src/Hosting.ts

```diff
@@ -200,6 +203,14 @@ export function Hosting<
       ];
     }
 
+    getNameField() : TextFieldBase {
+      return <TextFieldBase>this.shadowRoot!.querySelector('span[id="userField"]');
+    }
+
+    private onUserInfo(event: UserInfoEvent) {
+      this.getNameField().textContent = get('userinfo.loggedInAs', {name: event.detail.name});
```

Comment:
As far as I know you can also use "translate". That function is used the most.

Reply:
Adjusted!